android/src/main/java/com/ahm/capacitor/camera/preview/CameraPreview.java

```diff
@@ -205,6 +205,7 @@ private void startCamera(final PluginCall call) {
         final Integer width = call.getInt("width", 0);
         final Integer height = call.getInt("height", 0);
         final Integer paddingBottom = call.getInt("paddingBottom", 0);
+        final Boolean toBack = call.getBoolean("toBack", false);
```

Comment:
I would name it `placeToFront` and default would be false.
Cause other users of this library already expecting to have this positioned in the back, so it' will be a breaking change for them.

Reply:
Hmm I agree. But since this project is also similar to `cordova-plugin-camera-preview`. It's interface should also be consistent. https://github.com/capacitor-community/camera-preview/issues/49#issuecomment-684918007

Let's me see what other's think about this. Good point. Thanks for reviewing :+1:

Reply:
For what is my understanding, this plugin came to life to offer a replacement for [cordova-plugin-camera-preview](https://github.com/cordova-plugin-camera-preview/cordova-plugin-camera-preview) to anyone migrating from Cordova to Capacitor. Having this in mind, I think that having this plugin behaving like its Cordova counterpart is more important than having a (little) breaking change  My two cents.